spec/store_spec.rb

```diff
@@ -0,0 +1,75 @@
+RSpec.describe Store do
+  let (:aws_s3_client) { Aws::S3::Client.new(stub_responses: true) }
+  let (:s3_store) { Store.new }
+  
+  before do
+    allow(s3_store).to receive(:aws_client).and_return(aws_s3_client)
+  end
+  
+  context '#list_buckets' do
+    
+    it 'should list all the buckets in the linked account when response is not empty/nil' do
+      aws_s3_client.stub_responses(:list_buckets, buckets: [{ name: 'bucket1' }, { name: 'bucket2' }, { name: 'bucket3' }])
+      response = aws_s3_client.list_buckets
+      expect(response.buckets.count).to eq(3)
+      expect(STDOUT).to receive(:puts).thrice
```

Comment:
Add the actual value..
This would be better with proper logging rather than handle `puts`

Reply:
We need `puts` and not `logging` in this case. When we list buckets we want to display to the user a list of their buckets which would be through the `puts` statement. Logging with list out the bucket names in the log file which does not serve the purpose.

Reply:
So this will be taken up in the next story.